src/Message/Request.php

```diff
@@ -116,7 +134,9 @@ public function withPost(array $params)
             $body,
             $request->getProtocolVersion(),
             $request->getRetries(),
-            $request->getOptions()
+            $request->getOptions(),
+            $this->getServiceKey(),
```

Comment:
Shouldn't it be `$request->` ? The same below

Reply:
nope